src/styles/components/_card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  margin: auto;
+  padding: 3.2rem 1.6rem 1.6rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  transition: all 0.3s;
```

Comment:
It's not a good practice to use 'all' for the transition property. It's better to specify which properties you want to include in the transition. This is because 'all' can lead to performance issues, as it will trigger a transition for every property, even those that you don't need to transition.